src/main/java/COSE/ECPrivateKey.java

```diff
@@ -45,7 +45,7 @@ public ECPrivateKey(OneKey oneKey) throws CoseException, IOException
             System.arraycopy(X, 0, rgb, 1, X.length);
             rgb[0] = (byte) (2 + (oneKey.get(KeyKeys.EC2_Y).AsBoolean() ? 1 : 0));
             pubPoint = p.getCurve().decodePoint(rgb);
-            point = new ECPoint(point.getAffineX(), point.getAffineY());
+            point = new ECPoint(pubPoint.getAffineXCoord().toBigInteger(), pubPoint.getAffineYCoord().toBigInteger());
```

Comment:
There may be two problems here.  1) is this signed or unsigned 2) does ECPoint worry about the size of the value in terms of length?